### ISSUE TITLE
Display tooltip when hoovering over the legend

### ISSUE THREAD

**Issue by abrasat** [oxyplot/oxyplot#1628]
Is there any setting available, which allows displaying a custom tooltip when hoovering over the different legend entries?

**Comment by VisualMelon** [comment]
This is not presently possible, but it shouldn't be hard to implement if deemed useful. Adding a `LegendToolTip` to `Series` would probably suffice.

**Comment by abrasat** [comment]
Ok, thanks. I think this would be useful. I would like to put in the tooltip some description about each signal represented as line series.

**Comment by abrasat** [comment]
@VisualMelon : What happens with the "ToolTip" property from LineSeries right now, where is the content displayed?

**Comment by VisualMelon** [comment]
@abrasat that is displayed on mouse-over the series and the series entry in the tool tip. None of the tool-tips seem to be working for me at the moment... but that's what is meant to happen, at least.

**Comment by abrasat** [comment]
Yes, same here. No tooltip when moving mouse over any of the series.

**Comment by VisualMelon** [comment]
There are host of issues about tool tips: #1081 is probably still the case
